data/categories.txt

```diff
@@ -6851,6 +6978,10 @@ fi:Katu
 fr:Rue
 de:Straße
 hu:Utca
+fr:Rue
+de:Straße
```

Comment:
Why are these values duplicated?

Reply:
I honestly don't know, but they should all be removed now.